lib/nxdk/xbe.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Stefan Schmidt
+ *
+ * Licensed under the MIT License
+ */
+
+#ifndef __NXDK_XBE_H__
+#define __NXDK_XBE_H__
+
+#ifdef __cplusplus
+{
+#endif
+
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+void XGetXbePath (char *path, bool includeFileName);
```

Comment:
As indicated by my other comment regarding `XGetDashboardXbePath`, it would make more sense to be more explicit like `XGetCurrentXbePath()` (`CurrentXbe`? `RunningXbe`? `LoadedXbe`? `TitleXbe`? ...)

Reply:
Renamed to `XGetCurrentXbePath`.